tests/hikari/impl/test_event_manager.py

```diff
@@ -106,7 +105,7 @@ def event_manager(self, entity_factory, event_factory):
         return obj
 
     @pytest.fixture()
-    def stateless_event_manager(self, event_factory, entity_factory):
+    def stateless_event_manager_impl(self, event_factory, entity_factory):
```

Comment:
Renaming these fixtures which are referenced here instead of the imports adds a ton of noise to the diff in this pr which means ima put off reviewing the changes in this module cause its just a big "fuck you, review every line"

Reply:
Sorry, I had to rename `event_manager` to `event_manager_impl` since it shadowed the name. I also went ahead and did it for this one too for consistency sake.

Reply:
Welp if there's no other alternative and what I describe isn't doable then I'll just review this once i feel like reviewing a over 1k which gonna be a while tbh